messages/ulog/baro.yaml

```diff
@@ -0,0 +1,9 @@
+name: baro
+description: Contains barometer temperature, pressure and altitude
+fields:
+  - name: temp
+    type: float
+  - name: press
+    type: float
+  - name: alt
```

Comment:
Other values are raw sensor readings, but this is an estimated (and possibly filtered) altitude and should not be in the same message. In the future the altitude will be estimated from multiple sources as well.

Reply:
I even considered completely omitting altitude from the log, as it is currently redundant (directly derived from pressure), but kept it for the initial phase

We could create `AltEst` ULog message with `src` (estimation source), `est` (estimation) and `var` (variance) fields. Where altitude estimation source could be defined using `ULOG_AddInfo()`

Reply:
That would be good, it can be just called 'Altitude' since altitude is always estimated anyway (and again, no need to abbreviate that much).